config.py

```diff
@@ -8,8 +8,9 @@
 
 # Logging configuration
 LOGGING = {
-    "LEVEL": logging.DEBUG,
-    "HANDLER": logging.StreamHandler(sys.stdout)  # Log to stdout (see: https://12factor.net/logs)
+    "LEVEL": os.getenv("LOGGING_LEVEL"),
```

Comment:
If this is now an environment variable then is the README updated accordingly to handle this?
Why not leave it as a configuration for now?